extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -404,10 +404,16 @@ export class AmpA4A extends AMP.BaseElement {
       // onLayoutMeasure gets called multiple times.
       return;
     }
-    this.layoutMeasureExecuted_ = true;
+    const slotRect = this.getIntersectionElementLayoutBox();
+    if (slotRect.height == 0 || slotRect.width == 0) {
```

Comment:
I copied over the method the scheduler is using to decide whether to call layoutCallback to ensure parity with Delayed Fetch (see https://github.com/ampproject/amphtml/blob/master/src/service/resource.js#L614)